### PR TITLE
Default org view version export test

### PR DESCRIPTION
**Pull request by vision2910**
This PR is to add the functional test to cover the content view version export feature for default organization view.

### CODE REVIEW

File: test/functional/content_view/version/export_test.rb [hunk 1]
```diff
@@ -324,4 +324,58 @@
...
     assert_equal(HammerCLI::EX_SOFTWARE, result.exit_code)
   end
+
+  it "performs export for default organization view" do
+    params = [
+      '--id=1',
+      '--export-dir=/tmp/exports'
+    ]
+
+    ex = api_expects(:content_view_versions, :show)
```
Comment (akofink) [inline test/functional/content_view/version/export_test.rb]:
```suggestion
    ex = api_expects(:content_view_versions, :show).with_params('id' => '1')
```

File: test/functional/content_view/version/export_test.rb [hunk 2]
```diff
@@ -324,4 +324,58 @@
...
+      'id' => '1',
+      'repositories' => [{'id' => '2'}],
+      'major' => 1,
+      'minor' => 0,
+      'content_view' => {'name' => 'Default Organization View'},
+      'content_view_id' => 1,
+      'puppet_modules' => []
+    )
+
+    ex = api_expects(:content_views, :show)
```
Comment (akofink) [inline test/functional/content_view/version/export_test.rb]:
```suggestion
    ex = api_expects(:content_views, :show).with_params('id' => 3)
```

File: test/functional/content_view/version/export_test.rb [hunk 3]
```diff
@@ -324,4 +324,58 @@
...
+    ]
+
+    ex = api_expects(:content_view_versions, :show)
+    ex.returns(
+      'id' => '1',
+      'repositories' => [{'id' => '2'}],
+      'major' => 1,
+      'minor' => 0,
+      'content_view' => {'name' => 'Default Organization View'},
+      'content_view_id' => 1,
```
Comment (akofink) [inline test/functional/content_view/version/export_test.rb]:
```suggestion
      'content_view_id' => 3,
```

File: test/functional/content_view/version/export_test.rb [hunk 4]
```diff
@@ -324,4 +324,58 @@
...
+      'puppet_modules' => []
+    )
+
+    ex = api_expects(:content_views, :show)
+    ex.returns(
+      'id' => '1',
```
Comment (akofink) [inline test/functional/content_view/version/export_test.rb]:
```suggestion
      'id' => '3',
```

File: test/functional/content_view/version/export_test.rb [hunk 5]
```diff
@@ -324,4 +324,58 @@
...
+      'composite' => false,
+      'label' => 'Default_Organization_View'
+    )
+
+    ex = api_expects(:repositories, :show).with_params('id' => '2')
```
Comment (akofink) [inline test/functional/content_view/version/export_test.rb]:
```suggestion
    ex = api_expects(:repositories, :show).with_params('id' => '2')
```

File: test/functional/content_view/version/export_test.rb [hunk 6]
```diff
@@ -324,4 +324,58 @@
...
+    ex.returns(
+      'id' => '2',
+      'download_policy' => 'immediate',
+      'label' => 'Test_Repo',
+      'content_type' => 'yum',
+      'backend_identifier' => 'Default_Organization-Library-Test_Repo',
+      'relative_path' => 'Default_Organization/Library/Test_Repo',
+      'library_instance_id' => '1',
```
Comment (akofink) [inline test/functional/content_view/version/export_test.rb]:
```suggestion
      'library_instance_id' => '4',
```

File: test/functional/content_view/version/export_test.rb [hunk 7]
```diff
@@ -324,4 +324,58 @@
...
+      'content_counts' => {
+        'rpm' => 1,
+        'erratum' => 1
+      }
+    )
+
```
Comment (akofink) [inline test/functional/content_view/version/export_test.rb]:
```suggestion
    ex = api_expects(:repositories, :show).with_params('id' => '4')
    ex.returns(
      'id' => '4',
      'download_policy' => 'immediate',
      'label' => 'Test_Repo',
      'content_type' => 'yum',
      'backend_identifier' => 'Default_Organization-Library-Test_Repo',
      'relative_path' => 'Default_Organization/Library/Test_Repo',
      'content_counts' => {
        'rpm' => 1,
        'erratum' => 1
      }
    )

```

**Comment by vision2910** [comment]
@chris1984 no help needed atm, thanks. Had issues in dev env, almost fixed. Will update the changes..

**Comment by chris1984** [comment]
Closing out based on the big refactor of export in pr: 

https://github.com/Katello/hammer-cli-katello/commit/9bf7858adcf30410ae81ab9d430fd727eeb881bd

**Pull request closed without merging**